pkgs/top-level/all-packages.nix

```diff
@@ -38717,5 +38717,5 @@ with pkgs;
 
   oversteer = callPackage ../applications/misc/oversteer { };
 
-  input-leap = callPackage ../applications/misc/input-leap { };
+  input-leap = libsForQt5.callPackage ../applications/misc/input-leap { };
```

Comment:
The benefit of this is that we can do for example both Qt5 and Qt6 without having to change pkgs/applications/misc/input-leap/default.nix